### PR TITLE
test(ipfs): adds a test for ipfs with private entities PE-1987

### PR DESCRIPTION
**Pull request by matibat**
Changes
- Adds a unit tests to ensure --add-ipfs-flag and --private are disallowed in conjunction

### CODE REVIEW

File: src/CLICommand/parameters_helper.test.ts [hunk 1]
```diff
@@ -29,8 +29,7 @@ import {
 	MetaDataFileParameter,
 	MetaDataGqlTagsParameter,
 	MetadataJsonParameter,
-	IPFSParameter,
-	DataGqlTagsParameter
```
Comment (fedellen) [inline src/CLICommand/parameters_helper.test.ts:33]: Wonder why this is showing up here. DataGqlTags test was added in an adjacent PR not in PE-1880

**Review by fedellen — APPROVED** [review]
👏

File: src/CLICommand/parameters_helper.test.ts (hunk 1 above)
Comment (matibat) [reply src/CLICommand/parameters_helper.test.ts:33]: Yup, that was strange. It seems it was me the one that added that line c703ac2

File: src/CLICommand/cli_command.test.ts [hunk 2]
```diff
@@ -103,7 +114,10 @@ describe('CLICommand class', () => {
 				commandDescriptorForbiddenWalletFileAndSeedPhrase,
 				parsedCommandOptionsBothSpecified
 			);
-		}).to.throw();
+		}).to.throw('Parameter walletFile cannot be used in conjunction with seedPhrase');
```
Comment (karlprieb) [inline src/CLICommand/cli_command.test.ts:117]: 👏

**Review by karlprieb — APPROVED** [review]

**Pull request merged**